<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Performanta</class>
 <widget class="QDialog" name="Performanta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelmodificare">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>761</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pentru modificarea datelor unui angajat, selectati cu dublu click persoana dorita din lista de mai sus.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="salveaza">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>530</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salveaza si inchide</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adauga">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>490</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adauga un nou angajat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="salveaza2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>490</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salveaza</string>
   </property>
  </widget>
  <widget class="QLabel" name="info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Informatiile noului angajat, fiecare camp este obligatoriu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelnume">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nume:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelprenume">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prenume:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numeEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="prenumeEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="studii">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Se bifeaza daca angajatul are studii superioare</string>
   </property>
  </widget>
  <widget class="QLabel" name="hard">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Se bifeaza aptitudinile hardware pe care angajatul le detine.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hard1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Transport rutier</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hard2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Transport maritim</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hard3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Transport aerian</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hard4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Transport feroviar</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hard5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Transport international</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hard6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Transport domestic</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hard7">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Depozitare</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hard8">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Comercial</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hard9">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Achizitii</string>
   </property>
  </widget>
  <widget class="QLabel" name="soft">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Se bifeaza aptitudinile software pe care angajatul le detine.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="soft1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Management de echipa</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="soft2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Analiza date</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="soft3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Luare de decizii</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="soft4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Comunicare</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="soft5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lucru in echipa</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="soft6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prezentare a rezultatelor</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="soft7">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>280</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Initiative</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="soft8">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Angajament</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="soft9">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rezolvare a problemelor</string>
   </property>
  </widget>
  <widget class="QLabel" name="ap">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Se bifeaza aptitudinile formarile pe care angajatul le detine.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ap1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Managementul clientilor</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ap2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Managementul proiectelor</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ap3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Managementul timpului</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ap4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LEAN Management</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ap5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Centura SIX SIGMA</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ap6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Transport ADR</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ap7">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Transport IATA</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ap8">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>420</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sistem informatic</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ap9">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>450</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Standarde de lucru</string>
   </property>
  </widget>
  <widget class="QLabel" name="mhard">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nivel maturitate aptitudini hardware:</string>
   </property>
  </widget>
  <widget class="QLabel" name="msoft">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nivel maturitate aptitudini software:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hardEdit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="softEdit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="manager">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Se bifeaza daca angajatul este manager</string>
   </property>
  </widget>
  <widget class="QLabel" name="functie">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Functia anagajatului in cadrul agentiei:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="functieEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="agentie">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>470</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Agentia din care face parte:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>500</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
